projects/database-client-semconv.md

```diff
@@ -0,0 +1,38 @@
+# Database Client Semantic Convention Stability
+
+## Description
+
+Get the database client semantic conventions to stability, and afterwards update existing Database instrumentation across all OpenTelemetry repositories to conform with the stable conventions.
+
+## Deliverables
+
+* Mark the Database client semantic conventions as stable.
+* Update existing Database client instrumentation across all OpenTelemetry repositories to conform with the stable conventions.
+
+## Staffing / Help Wanted
+
+* @trask project lead
+* @lmolkova domain expert
+* TBD more domain experts
+* TBD technical committee sponsor
```

Comment:
I volunteer as a TC sponsor.

I can also help prototype in java.

I'm committed to attend quite a few meetings already, so may not be able to make two meetings a week, but can commit to attending at least one per week.